signalapi/signal.go

```diff
@@ -74,11 +80,46 @@ func (h *Handler) ServeCreateSignals(w http.ResponseWriter, r *http.Request) {
 	signals := []signal.Signal{}
 
 	for _, rule := range rules {
-		if rule.SendAlert {
-			// TODO: implement create alert logic
-			continue
-		}
 		for _, action := range rule.Actions {
+			if rule.SendAlert && action.DestType == "goalert" {
```

Comment:
```suggestion
			if rule.SendAlert && strings.EqualFold(action.DestType, "alert") {
```
Latest UI code is setting destType as 'ALERT' when 'Create Alert' is toggled. `strings.EqualFold` for case-insensitivity.